waiverdb/templates/permissions.html

```diff
@@ -14,28 +14,25 @@
                         <th>Name</th>
                         <th>Maintainers</th>
                         <th>Testcases</th>
-                        <th>Users/Groups</th>
+                        <th>Groups</th>
+                        <th>Users</th>
+                        <th>Description</th>
                     </tr>
                 </thead>
                 <tbody>
                     {% for p in permissions -%}
                     <tr>
                         <td>{{ p["name"] }}</td>
-                        <td>{{ p["maintainers"] | map("urlize") | join(" ") | safe }}</td>
-                        <td>{{ p["testcases"] | join("<br />") | replace('-', '&#8209;') | safe }}</td>
                         <td>
-                            {% if p["groups"] | length > 0 -%}
-                            <span class="font-weight-bold">Groups:</span><br />
-                            {{ p["groups"] | join("<br />") | replace('-', '&#8209;') | safe }}
-                                {% if p["users"] | length > 0 -%}
-                                <br />
-                                {%- endif %}
-                            {%- endif %}
-                            {% if p["users"] | length > 0 -%}
-                            <span class="font-weight-bold">Users:</span><br />
-                            {{ p["users"] | join("<br />") | replace('-', '&#8209;') | safe }}
-                            {%- endif %}
+                          {% for maintainer in p["maintainers"] %}<div>{{ maintainer | urlize }}</div>{% endfor %}
                         </td>
+                        <td>
+                          <div>{{ p["_testcase_regex_pattern"] }}</div>
+                          {% for testcase in p["testcases"] %}<div>{{ testcase }}</div>{% endfor %}
```

Comment:
It's generally not a good idea to put a block-level element `div` inside a `td`, cause both of them define different way of content layout. In this case you can use `p` instead.

Reply:
Fixed by using `<il>` in unordered lists.